core/domain/html_cleaner.py

```diff
@@ -136,6 +137,10 @@ def get_image_filenames_from_html_strings(html_strings):
         if 'id' in rte_comp and rte_comp['id'] == 'oppia-noninteractive-image':
             filenames.append(
                 rte_comp['customization_args']['filepath-with-value'])
+        elif ('id' in rte_comp and
+              rte_comp['id'] == 'oppia-noninteractive-svgeditor'):
```

Comment:
You should not really be naming this component an "svg editor". It's not an editor. It's an SVG image, or an SVG diagram, or similar.

Reply:
Done.